src/handlebar-helpers.js

```diff
@@ -0,0 +1,26 @@
+// list of useful handlebars helpers we can bundle
+// credit goes to https://stackoverflow.com/a/31632215 & https://stackoverflow.com/a/50427569
+//
+// TODO: maybe make this extensible via another CR field?
+const helpers = {
+  assign: function (varName, varValue, options) {
+    if (!options.data.root) {
+      options.data.root = {};
+    }
+    options.data.root[varName] = varValue;
```

Comment:
im not sure how concrete this data object is. lets use const objectPath = require('object-path'); to make sure we dont hit any errors

Reply:
updated